shell/assets/translations/en-us.yaml

```diff
@@ -1858,9 +1858,15 @@ cluster:
       podAffinity: Affinity
       podTolerations: Tolerations
       podRequestsAndLimits: Requests and Limits
+      schedulingCustomization: Scheduling Customization
     subGroups:
       podAffinityAnti: Pod Affinity/Anti-Affinity
       nodeAffinity: Node Affinity
+      schedulingCustomization: 
+        label: Prevent Rancher cluster agent pod eviction
+        description: This ensures that the cluster agent pod is not evicted due to node pressure, resource constraints or scheduler policies.
+        banner: The setting of this feature in this cluster are different from the global settings.
```

Comment:
This grammar is not quite right. It should be something like

`This feature has been enabled with settings that are different from the global default settings.`